angular-workspace/projects/ni/nimble-angular/src/thirdparty/directives/radio_control_value_accessor.ts

```diff
@@ -0,0 +1,225 @@
+ * - Commented out now-unused throwNameError() function
+ * - Removed now-unused import for RuntimeErrorCode and RuntimeError
+ * - Updated import of NgControl to pull from package export
+ */
+
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Directive, ElementRef, forwardRef, Injectable, Injector, Input, NgModule, OnDestroy, OnInit, Renderer2} from '@angular/core';
+
+import {BuiltInControlValueAccessor, ControlValueAccessor, NG_VALUE_ACCESSOR} from './control_value_accessor';
+import {NgControl} from '@angular/forms';
+
+export const RADIO_VALUE_ACCESSOR: any = {
+  provide: NG_VALUE_ACCESSOR,
+  useExisting: forwardRef(() => RadioControlValueAccessor),
+  multi: true
+};
+
+/* [Nimble] Commenting because unused
+function throwNameError() {
+  throw new RuntimeError(RuntimeErrorCode.NAME_AND_FORM_CONTROL_NAME_MUST_MATCH, `
+      If you define both a name and a formControlName attribute on your radio button, their values
+      must match. Ex: <input type="radio" formControlName="food" name="food">
+    `);
+}
+*/
+
+/**
+ * Internal-only NgModule that works as a host for the `RadioControlRegistry` tree-shakable
+ * provider. Note: the `InternalFormsSharedModule` can not be used here directly, since it's
+ * declared *after* the `RadioControlRegistry` class and the `providedIn` doesn't support
+ * `forwardRef` logic.
+ */
+@NgModule()
+export class RadioControlRegistryModule {
+}
+
+/**
+ * @description
+ * Class used by Angular to track radio buttons. For internal use only.
+ */
+@Injectable({providedIn: RadioControlRegistryModule})
+export class RadioControlRegistry {
+  private _accessors: any[] = [];
+
+  /**
+   * @description
+   * Adds a control to the internal registry. For internal use only.
+   */
+  add(control: NgControl, accessor: RadioControlValueAccessor) {
+    this._accessors.push([control, accessor]);
+  }
+
+  /**
+   * @description
+   * Removes a control from the internal registry. For internal use only.
+   */
+  remove(accessor: RadioControlValueAccessor) {
+    for (let i = this._accessors.length - 1; i >= 0; --i) {
+      if (this._accessors[i][1] === accessor) {
+        this._accessors.splice(i, 1);
+        return;
+      }
+    }
+  }
+
+  /**
+   * @description
+   * Selects a radio button. For internal use only.
+   */
+  select(accessor: RadioControlValueAccessor) {
+    this._accessors.forEach((c) => {
+      if (this._isSameGroup(c, accessor) && c[1] !== accessor) {
+        c[1].fireUncheck(accessor.value);
+      }
+    });
+  }
+
+  private _isSameGroup(
+      controlPair: [NgControl, RadioControlValueAccessor],
+      accessor: RadioControlValueAccessor): boolean {
+    if (!controlPair[0].control) return false;
+    // @ts-ignore: Use of internal NgControl member _parent
```

Comment:
since this is our comment we should include `[Nimble]` and `ts-expect-error` would be more appropriate (the linter would normally require it but isn't enabled here as expected)

```suggestion
    // @ts-expect-error: [Nimble] Use of internal NgControl member _parent
```